imports/api/methods.js

```diff
@@ -31,5 +31,16 @@ Meteor.methods({
   },
   findCompanyByJob: function (id) {
     return Jobs.find({_id: id}).fetch()[0].company;
+  },
+  applyToJob: function(job) {
+    let userId = Meteor.userId();
+    
+    Jobs.update(job, {
+      $push: { applications: userId }}, {getAutoValues: false}
+    );
+
+    Meteor.users.update(userId, {
+      $push: { "profile.applications": job }}, {getAutoValues: false}
+    );
```

Comment:
That's the method responsible for updating the User and Job schemas and telling the user has applied for the opportunity.

In case you are wondering what the heck is `{getAutoValues: false}`, this is to avoid updating the fields that are autoValues, such as **createdAt**, otherwise everytime we update the schema will have a new **createdAt** date.